### PR TITLE
less go versions are tested because travis breaks for go 1.5.4 and go…

### PR DESCRIPTION
**Pull request by awalterschulze**
… 1.6.1, even on old commits and reverted commits which worked before

### CODE REVIEW

**Comment by tamird** [comment]
You should make these allowed failures rather than removing them entirely.

**Comment by awalterschulze** [comment]
Is there a way to do that in travis?

**Comment by awalterschulze** [comment]
Lets see what happens 
http://stackoverflow.com/questions/40919256/older-go-versions-on-travis-are-killed-in-the-middle-of-running-tests

**Comment by tamird** [comment]
https://docs.travis-ci.com/user/customizing-the-build#Rows-that-are-Allowed-to-Fail

**Comment by awalterschulze** [comment]
We can bring these tests back in a pull request, once they pass

**Comment by awalterschulze** [comment]
Hmmm, ok but we aren't using matrix
My travis knowledge is pretty weak

**Comment by awalterschulze** [comment]
Ok I think I figured it out.  Lets see what happens

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +15,11 @@ script:
 language: go
 
 go:
-  - 1.5.4
-  - 1.6.3
```
Comment (tamird) [inline .travis.yml]: you should probably restore this bit and squash your commits before merging.

Comment (awalterschulze) [inline .travis.yml]: fair :)

**Comment by awalterschulze** [comment]
Seems like its still failing, even though the fails are allowed.

File: .travis.yml [hunk 2]
```diff
@@ -15,6 +15,11 @@ script:
...
   - 1.7.1
+  - 1.6.3
+  - 1.5.4
+
+matrix:
+  allow_failures:
+    - 1.6.3
```
Comment (tamird) [inline .travis.yml]: these need to be `- go: 1.6.3`

Comment (awalterschulze) [inline .travis.yml]: Thanks.

**Comment by awalterschulze** [comment]
Looks like its going to work :)

**Comment by awalterschulze** [comment]
Awesome thanks a lot :)

**Pull request merged**